environments/network/demo.tfvars

```diff
@@ -0,0 +1,49 @@
+enable_debug = "true"
+
+network_address_space                  = "10.50.64.0/18"
+aks_00_subnet_cidr_blocks              = "10.50.64.0/20"
+aks_01_subnet_cidr_blocks              = "10.50.80.0/20"
+iaas_subnet_cidr_blocks                = "10.50.96.0/24"
+application_gateway_subnet_cidr_blocks = "10.50.97.0/25"
+
+additional_subnets = [
+]
+
+private_dns_subscription = "1baf5470-1c3e-40d3-a6f7-74bfbce4b348"
+private_dns_zones = [
+  "demo.platform.hmcts.net",
+  "service.core-compute-demo.internal",
+  "service.core-compute-idam-demo.internal"
+]
+
+hub = "nonprod"
+
+additional_routes = [
+  {
+    name                   = "172_16_0_0"
```

Comment:
This one is needed as Nicks work for the availability zones is in the main branch.

Reply:
routing updated. Also added count to aks-network-module cft branch that the network component uses so that the default route is not deployed for the demo env.